_includes/1.0/api/json/purchase_order.json

```diff
@@ -0,0 +1,87 @@
+{
+  "id": 10001,
+  "uuid": "f67691ec-fa87-4971-b300-c0b0ca1d1015",
+  "vendor": {
```

Comment:
I wasn't aware that the Purchase order was returning the whole vendor.

Can you change the back end to use a `NameUserCodePOJO` instead, like it's done for Invoice customer ? And then update the documentation.